panel/panel.py

```diff
@@ -9,17 +9,27 @@
     "auto_indent": False,
     "draw_indent_guides": False,
     "draw_white_space": "None",
-    "gutter": False,
+    "gutter": True,
     "is_widget": True,
     "line_numbers": False,
     "margin": 3,
     "match_brackets": False,
+    # path/file.ext:
+    "result_file_regex": r"^(.*):$",
+    # type  row:col  linter  code  message
+    # <col> is optional
+    "result_line_regex": r"^\s+[⨻⨂]\s+(\d+)(?::(\d+))?\s+.*$",
     "scroll_past_end": False,
-    "tab_size": 4,
+    "tab_size": 2,
```

Comment:
Does this even matter? I'm a 4-spaces guy usually..

Reply:
Me, too. Vouch for 4-spaces.